mne/conftest.py

```diff
@@ -1172,3 +1172,53 @@ def pytest_runtest_makereport(item, call):
     outcome = yield
     rep = outcome.get_result()
     item.stash.setdefault(_phase_report_key, {})[rep.when] = rep
+
+
+@pytest.fixture(scope="function")
+def eyetrack_cal(request):
```

Comment:
No need to take `request` fixture here because you don't use it
```suggestion
def eyetrack_cal():
```